float64[] inputs
float64 max_F_xr
float64 max_F_xf
float64 max_F_yr
float64 max_F_yf
float64 t_execution
float64 dx
float64 dy
float64 du